web/wp-content/themes/dctx-v2/template-parts/content-blocks/block-wds-fifty-media-text.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-bg7blw.php, but found phpcs-scan-BG7blW.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).